rpcserver.go

```diff
@@ -2446,6 +2446,13 @@ func (r *rpcServer) AddInvoice(ctx context.Context,
 	}
 
 	amt := btcutil.Amount(invoice.Value)
+	
+	// The value of the invoice can not be negative
```

Comment:
Per the contribution guideline, [all comments should be full sentences](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#43-code-documentation-and-commenting).